### ISSUE TITLE
Glitched rendering

### ISSUE THREAD

**Issue by i0x0** [xCollateral/VulkanMod#165]
I don't know another way to describe this other than rendered noise
![Screenshot_20221226_023615](https://user-images.githubusercontent.com/29006605/209519256-a51c89bb-cfbd-45c8-97f4-e56a22e682fa.png)
![2022-12-26_02 19 06](https://user-images.githubusercontent.com/29006605/209519286-f00a5347-512b-44a4-ae02-6097f5b6e90e.png)

**Comment by i0x0** [comment]
Now looking at the screenshot I took in Minecraft it looks completely different than what was actually being shown
![Screenshot_20221226_024015](https://user-images.githubusercontent.com/29006605/209519548-4ea4c559-48ab-4eec-917f-c7d52cdd7591.png)

**Comment by xCollateral** [comment]
What are your specs and OS? I guess you're probably on intel gpu.

**Comment by Connor-GH** [comment]
Their OS is linux. You can tell from the KDE desktop environment.

**Comment by Connor-GH** [comment]
I am also on linux and experienced some glitched rendering on 1.18. There was very shaky/stuttery movement for about 4 seconds, I then pressed F3, and about two seconds later my screen froze, went black, came back on with a corrupted picture, then gave no output to the monitor (at this point the monitor says "no signal"). I actually had to restart my computer. 

It is a very different kind of glitched rendering, I'd like to say.

specs are as follows:
```
OS: Gentoo Linux x86_64
Kernel: 5.15.85-gentoo-dist-custom
Resolution: 3840x2160 @ 59.98Hz
WM: sway (wlroots)
CPU: AMD Ryzen 7 5700G
GPU: Radeon RX 6600
```

Latest version of mesa available (22.3.1)
vulkan driver is RADV, bundled with mesa.

No mods other than fabric API were used.

**Comment by nm17** [comment]
Can't reproduce this issue on my Debian bookworm setup (NVIDIA GeForce GTX 1650) on [0.2.0 build 2](https://modrinth.com/mod/vulkanmod/version/0.2.0+2) (which is the latest available version of VulkanMod on Modrinth for MC 1.19.2 as of the time of this issue's creation, which I assume OP used), but my screenshot of the game does look like the one in OP's issue description ( https://github.com/xCollateral/VulkanMod/issues/165#issue-1510703220 ). In game and in menus everything is fine.

![2023-03-28_19 53 08](https://user-images.githubusercontent.com/23419131/228297775-8aa10c65-cdf3-4170-bb74-a4e3253ab0ca.png)

On VulkanMod version 0.2.5 for 1.19.2 everything is fine, including the screenshot.

![2023-03-28_19 56 09](https://user-images.githubusercontent.com/23419131/228299908-562b83b2-a2ce-46bb-8fcf-cbe1f85b7895.png)

@i0x0 , can you confirm which version of VulkanMod you used and if the bug persists on 0.2.5?

**Comment by nm17** [comment]
The blueish screenshot issue seems to be https://github.com/xCollateral/VulkanMod/issues/151 , which was fixed in 12818cde13de2aa6562a6a528a2ad1e3544cf0ee

**Comment by xCollateral** [comment]
It seems fixed, If that's not the case let me know.
